sklearn/decomposition/tests/test_pca.py

```diff
@@ -529,6 +529,26 @@ def test_pca_score3():
     assert_true(ll.argmax() == 1)
 
 
+def test_pca_score4():
```

Comment:
@lesteve Thanks. But I think it may not be suitable since the small eigenvalues are usually very small. Their sum may still be smaller than the big eigenvalues, making it not a good regression test.